app/src/main/res/values/strings.xml

```diff
@@ -204,6 +204,8 @@
     <string name="general_settings">General Settings</string>
     <string name="high_value">High Value</string>
     <string name="low_value">Low Value</string>
+    <string name="stats_high_value">High Value: </string>
```

Comment:
This duplicates the resource strings and increases the translation effort. We should think about different solutions, i. e. the [Android Stem](https://github.com/LikeTheSalad/android-stem) Gradle plugin. And example usage is shown at https://stackoverflow.com/a/59767841.

Reply:
I wanted to append a colon to the existing strings "High Value" and "Low Value".  But, I didn't know how to do it easily.
Assuming this is merged, I hope some time in the future to propose a second step for this where instead of using the existing High and Low values to change these to be inputs so that the user can specify what they want to use for their statistical analysis not necessarily the same as the values used for their alarms.
But, I didn't want to make this a very large PR.  So, that's a second step.

So, in the future, we will need separate strings with respect to the existing high and low values.